### ISSUE TITLE
PHP warning on ImageAccessibility checker

### ISSUE THREAD

**Issue by oburk** [p1ho/accessibility-checker#3]
Hi, 

I found a bug in regular expression used by function `_eval_imgs`. When you use the accessibility-checker like this:

```php

        $colorContrastChecker = new ColorContrastChecker();
        $headingStructureChecker = new HeadingStructureChecker();
        $imageAccessibilityChecker = new ImageAccessibilityChecker();

            $dom = new DOMDocument();
            $dom->loadHTML($domString);

            $results = [
                $colorContrastChecker->evaluate($dom),
                $headingStructureChecker->evaluate($dom),
                $imageAccessibilityChecker->evaluate($dom),
            ];

```
using this DOM

```html
<img id="img_1694" onclick="" class="mediaListPreviewImg" src="/medialibrary//images/Hydrangeas_5_thn.jpg?t=1582622719" alt="Edit Inactive Image File">'
```

you got the following PHP warning

```
Warning: preg_match(): Compilation failed: invalid range in character class at offset 7 in \vendor\p1ho\accessibility-checker\src\ImageAccessibility\Checker.php on line 88
```

I opened a merge  #2 which can solve the warning.

**Comment by p1ho** [comment]
Thanks! I'm currently travelling, so can't test it right now. But I'll test it out tomorrow to see if I can reproduce it.

**Comment by p1ho** [comment]
~Hello, I'm unable to produce the issue on my machine, which PHP version are you running?~
[Similar problem with WordPress](https://wordpress.org/support/topic/php-warning-preg_match-compilation-failed/)

Edit: Was able to reproduce it with [PHP 7.3](https://travis-ci.com/p1ho/accessibility-checker/jobs/291826086)
